code/game/objects/items/weapons/tape.dm

```diff
@@ -40,6 +40,28 @@
 		qdel(src)
 
 
+/obj/item/stack/tape_roll/afterattack(atom/target, mob/user, proximity)
```

Comment:
Good heavens, we have procs for using tape that makes 99% of this redundant. Heaven's almighty.